README.md

```diff
@@ -69,7 +69,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thanks @HAMMAS-SALEEM, I'll look into it